### ISSUE TITLE
--update appears to succeed then hangs

### ISSUE THREAD

**Issue by tscheffe** [tldr-pages/tldr-node-client#19]
Ran into an issue when I was trying to get up and running.
```sh
=> tldr --update
Updating...
Done
|                <--- Cursor remains here indefinitely
```

Used `ctrl + c` to quit and then tried to run `tldr --update` again and got this:
```sh
=> tldr --update
Updating...
Error: invalid signature: 0xa0000
    at /usr/local/lib/node_modules/tldr/node_modules/unzip2/lib/parse.js:63:13
    at processImmediate [as _immediateCallback] (timers.js:374:17)
```
which is discussed in Issue #12.

I downloaded version 1.01 via npm today.

Thoughts?

**Comment by tscheffe** [comment]
Additional Note:
I just ran `tldr --clear-cache` and then `tldr --update` and it succeeded. Maybe it only fails on a fresh install?

**Comment by rprieto** [comment]
Thanks @tscheffe, I just ran into the `invalid signature` issue several times in a row, and `--clear-cache` fixed it for me to. I think you might be onto something!

**Comment by igorshubovych** [comment]
This should be resolved after #21 merged. We will soon release new version.
